lib/pos_snapshot_entries.go

```diff
@@ -15,13 +15,17 @@ import (
 
 const SnapshotLookbackNumEpochs uint64 = 2
 
-func (bav *UtxoView) GetSnapshotEpochNumber() (uint64, error) {
+func (bav *UtxoView) GetCurrentSnapshotEpochNumber() (uint64, error) {
 	// Retrieve the CurrentEpochNumber.
 	currentEpochNumber, err := bav.GetCurrentEpochNumber()
 	if err != nil {
-		return 0, errors.Wrapf(err, "GetSnapshotEpochNumber: problem retrieving CurrentEpochNumber: ")
+		return 0, errors.Wrapf(err, "GetCurrentSnapshotEpochNumber: problem retrieving CurrentEpochNumber: ")
 	}
-	if currentEpochNumber < SnapshotLookbackNumEpochs {
+	return bav.GetSnapshotEpochNumber(currentEpochNumber)
+}
+
+func (bav *UtxoView) GetSnapshotEpochNumber(epochNumber uint64) (uint64, error) {
```

Comment:
nit: it's worth clarifying a bit further that this function just maps a given epoch number to its snapshot epoch enumber
```suggestion
func (bav *UtxoView) ComputeSnapshotEpochNumberForEpoch(epochNumber uint64) (uint64, error) {
```